Documentation/operations/system_requirements.rst

```diff
@@ -63,6 +63,7 @@ Debian_                    >= 9 Stretch
 `Fedora Atomic/Core`_      >= 25
 Flatcar_                   all
 LinuxKit_                  all
+Raspbian_                  [#raspbian_foot]_
```

Comment:
1. Now, the official name of Raspbian is Raspberry Pi OS (https://www.raspberrypi.com/software/). Could you change it?
2. With which version of Raspberry Pi OS did you test? Could you write it to the `Minimum Version` column?
3. When I preview this `Raspbian`, it is displayed like `Raspbian_` (unnecessary underscore). Could you fix that?

Reply:
Regarding 3., this is because the link for the `Raspbian` target is missing in the list below. See how all other distributions have a link to their website, just below the array. For Raspberry Pi OS, I guess it would be <https://www.raspberrypi.com/software/>? Adding the link will also fix the build issues reported by the CI.